package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-utils-formattingutils",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
Yes, it's  breaking compatibility changes. You can change your code to use this library. So, the version was increased to 2.0.0